cognee/cli/tui/app.py

```
@@ -0,0 +1,129 @@
+        align: center middle;
+    }
+    """
+    
+    TITLE = "Cognee TUI - Knowledge Graph Manager"
+    SUB_TITLE = "Navigate with arrow keys • Press ? for help"
+    
+    BINDINGS = [
+        Binding("q", "quit", "Quit", priority=True),
+        Binding("?", "help", "Help"),
+        Binding("d", "toggle_dark", "Toggle Dark Mode"),
+    ]
```

Ruff (0.14.4)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)